src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -69,6 +77,7 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
   const [isAuthenticationModalOpen, setIsAuthenticationModalOpen] = useState(
     false
   );
+  const [isJoinVenueModalOpen, setIsJoinVenueModalOpen] = useState(true);
```

Comment:
I often name these as follows (isX, setX):

```suggestion
  const [isJoinVenueModalOpen, setJoinVenueModalOpen] = useState(true);
```